src/configure.rs

```diff
@@ -102,9 +100,72 @@ pub fn print_configuration(use_default: bool) {
                 .collect::<String>()
         );
     }
+
+    let print_config = if paths.is_empty() {
+        config
+    } else {
+        extract_toml_paths(config, paths)
+    };
+
+    let string_config = toml::to_string_pretty(&print_config).unwrap();
+
     println!("{}", string_config);
 }
 
+fn extract_toml_paths(mut config: toml::Value, paths: &[&str]) -> toml::Value {
```

Comment:
I think it might be simpler to remove any values from the config that are not listed in `paths` instead of copying them over, but this also looks good to me.

Reply:
To do it that way, the code would need to recursively compare the keys that are present to the provided paths. I guess if the paths were parsed into a recursive structure first it wouldn't be too bad, but I think it would be significantly more complicated.